python-template/{{cookiecutter.placeholder_repo_name}}/docs/decisions/0001-purpose-of-this-repo.rst

```diff
@@ -0,0 +1,27 @@
+1. Purpose of this Repo
+=======================
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+TODO: Give context on what lead to the creation of this repo. What project is the repo related to?
+
+Decision
+--------
+
+TODO
```

Comment:
I could not figure out what instructions to give here.

Reply:
`TODO: Clearly state how the context above leads you to creating this repo.`